cpp/src/slice2swift/Gen.cpp

```diff
@@ -25,6 +26,28 @@ namespace
         assert(dc);
         return dc->getMetadataArgs("swift:class-resolver-prefix").value_or("");
     }
+
```

Comment:
This function was copy-pasted from `SwiftUtil.cpp`, since it is no longer used in that file.